### ISSUE TITLE
schema validation for consumes property when using Swagger file type needs to be updated.

### ISSUE THREAD

**Issue by JoyPeterson** [krakenjs/swaggerize-routes#58]
At the beginning of validator.js where the types var is declared, the value of the consumes property is declared as

```javascript
                consumes: {
                    type: 'string',
                    pattern: /multipart\/form-data|application\/x-www-form-urlencoded/,
                    required: true
                },
```

However, with the current version of Swagger, consumes is coming in as an array. e.g. 

```javascript
[ 'multipart/form-data' ]
```

I think the schema for consumes needs to be updated to 

```javascript
                consumes: {
                    type: 'array',
                    required: true,
                    items: {
                        type: 'string',
                        pattern: /multipart\/form-data|application\/x-www-form-urlencoded/
                    },
                    minItems: 1,
                    uniqueItems: true
                },
```

This appears to be working correctly in my limited testing, but I am fairly new to Swagger so I might be missing something.

If backwards compatibility is desired, it could be updated to:

```javascript
                consumes: {
                    oneOf: [
                        {
                            type: 'array',
                            required: true,
                            items: {
                                type: 'string',
                                pattern: /multipart\/form-data|application\/x-www-form-urlencoded/
                            },
                            minItems: 1,
                            uniqueItems: true
                        },
                        {
                            type: 'string',
                            pattern: /multipart\/form-data|application\/x-www-form-urlencoded/,
                            required: true
                        }
                    ]
                },
```

**Comment by JoyPeterson** [comment]
I now see that this was fixed by pull request #55.  I will submit a pull request to update the unit tests in order to prevent a regression.

**Comment by tlivings** [comment]
Resolved, thanks.
